index.html

```diff
@@ -4,63 +4,219 @@
+                        <img src="./assets/img-block1/Frame1.png" alt="">   
+                    </div>
+                    <p class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <nav>
+                        <ul>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/html.png" alt="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/css.png" alt="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/javascript.png" alt="">
+                                </a>
+                            </li>
+                        </ul>
+                    </nav>
+                    <div>
+                        <button type="button" class="see_button">See Project</button>
+                    </div>
+                </div>                
+            </section>
+<!--about myself section from here-->
+            <div id="myself_div">    
+                <section class="picture_container">
+                    <div class="myself">   
+                        <h1 class="title">About<br>Myself</h1>
+                        <p class="content">Hello I'm a software developer! I can help you build a product , 
+                        feature or website Look through some of my work and experience! 
+                        If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+
+                        <p class="lets">LET'S CONNECT</p>
 
-          <p class="lets">LET'S CONNECT</p>
-          
-          <nav>
-            <ul>
-              <li>
-                <a href="" target="_blank">
-                  <img src="./assets/icons/Twiter.svg" alt=""/>
-                </a>
-              </li>  
-              <li>
-                <a href="" target="_blank">
-                  <img src="./assets/icons/Linkedin_icon.png" alt=""/>
-                </a>
-              </li>
-              <li>  
-                <a href="" target="_blank">
-                  <img src="./assets/icons/M.svg" alt=""/>
-                </a>
-              </li>
-              <li>  
-                <a href="" target="_blank">
-                  <img src="./assets/icons/Github.svg" alt=""/>
-                </a>
-              </li>
-              <li> 
-                <a href="" target="_blank">
-                  <img src="./assets/icons/Fin.svg" alt=""/>
-                </a>
-              </li>  
-            </ul>  
-          </nav>
-        </section>
+                        <nav>
```

Comment:
- [ ] Kindly remove the `nav` element since these are not navigation links. This element is reserved for the main navigation links in your website for semantic reasons.